include/aspect/melt.h

```diff
@@ -137,132 +137,132 @@ namespace aspect
   namespace Assemblers
   {
     /**
-     * A class for the definition of functions that implement the
-     * linear system terms for the *melt* migration compressible or
-     * incompressible equations.
-     */
+      * A base class for the definition of assemblers that implement the
+      * linear system terms for the *melt* migration compressible or
+      * incompressible equations.
+      */
     template <int dim>
-    class MeltEquations : public aspect::internal::Assembly::Assemblers::AssemblerBase<dim>,
+    class MeltInterface : public aspect::Assemblers::Interface<dim>,
       public SimulatorAccess<dim>
     {
       public:
+        virtual ~MeltInterface () {};
```

Comment:
I don't think you need the destructor here as this is still an abstract base class